pkg/didcomm/protocol/outofband/service.go

```diff
@@ -31,6 +31,8 @@ const (
 	RequestMsgType = "https://didcomm.org/oob-request/1.0/request"
 	// InvitationMsgType is the '@type' for the invitation message.
 	InvitationMsgType = "https://didcomm.org/out-of-band/1.0/invitation"
+	// OldInvitationMsgType is the '@type' for the old invitation message
+	OldInvitationMsgType = "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/out-of-band/1.0/invitation"
```

Comment:
https://github.com/hyperledger/aries-framework-go/pull/2655#discussion_r598115168

Reply:
Seems like this was replaced in the DIDComm service on a higher level but that didn't seem to pass on for the OOB protocol

Reply:
ok